.github/workflows/trello.yml

```diff
@@ -0,0 +1,20 @@
+name: GitHub Commit To Trello Comment
+
+on: [push, pull_request]
```

Comment:
I think, that this file is not related to the current PR. Why it here?

Reply:
I chose the wrong base to merge. Fixed